app/views/estimates/check_answers.html.slim

```diff
@@ -1,5 +1,5 @@
 - content_for :back
-  = back_link(:check_answers, @estimate, @back_buttons_invoke_browser_back_behaviour)
+  = back_link(:check_answers, @estimate, true)
```

Comment:
My instinct is that this back link on this screen should always go back to the last wizard step screen (which happens to always be the assets screen). By using the browser back behaviour, once the user has gone through a mini-loop, clicking back will take them to the final screen of the mini-loop, which feels wrong to me. Happy to be told I'm wrong if you disagree.

Reply:
I don't have strong feelings on this myself as both suggestions make sense, however govuk indicates that it should go back to the previous page:
https://design-system.service.gov.uk/components/back-link/
 >>Make sure the link takes users to the previous page they were on, in the state they last saw it. Where possible, ensure it works even when JavaScript is not available.<<

Reply:
Ok, I'm happy to accept the govuk official line on this.